backend/user-service/src/models/user.repository.ts

```diff
@@ -32,6 +32,6 @@ export async function updateUser(id: string, dto: UserDto | UserProfileDto): Pro
     return userModel.findByIdAndUpdate(id, dto, { new: true })
 }
 
-export async function deleteUser(id: string): Promise<void> {
-    await userModel.findByIdAndDelete(id)
+export async function deleteUser(id: string): Promise<IUser | null> {
+    return userModel.findByIdAndDelete(id)
```

Comment:
do we need to return the deleted user?

Reply:
not really i guess, it was more for testing purposes and verification